src/components/PublicProfile.vue

```diff
@@ -42,17 +102,27 @@
 <script>
   import db from '../database';
   import util from '../util/util';
+  import firebase from 'firebase';
+  import AuthMixin from '../mixins/auth'
```

Comment:
It doesn't look like you need to important the AuthMixin either?

Reply:
WE do need this for current user